venue.md

```diff
@@ -2,26 +2,12 @@
 title: Venue
 ---
 
-LambdAle will be held in the ballroom at [Balham Bowls Club](http://balhambowlsclub.com/).
+LambdAle will be held in the ballroom at [The George IV Pub](https://www.georgeiv.co.uk).
```

Comment:
s/ballroom/Boston Room

Reply:
I'm not sure what this means - couldn't find a link to the specific room on their website.

Reply:
Their largest function room is called the Boston Room. There's a reference to it on the ["360 tour" page](https://www.georgeiv.co.uk/360-tour).